### DISCUSSION TITLE
Where would the total team information submission form be given?

### DISCUSSION THREAD

**Discussion by SadmanIslamAlif** [larics/icuas22_competition#32] [Q&A] [answered]
Hello and Greetings, 

I could not find the form where I had to submit my total team information. Where it would be given? 

Thank you 

Regards 
Sadman 
President 
Robotics Club of BRAC University

**Comment by fpetric** [comment, marked as answer]
Hi @SadmanIslamAlif, 

thanks for reaching out. We have a few more teams than we expected, so we are currently upgrading the infrastructure for submission of both the application forms and your solutions. We will announce details soon.

**Reply by afaroo01** [reply]
Hi @fpetric, hope you are doing good.  Want to ask, until when the final application forms be received by the team to register officially? I read that the evaluation period was extended and will begin from April 1, but we haven't the submission form instructions. Will both be done at the same time?

**Reply by fpetric** [reply]
Hi :) the deadline for team documentation is April 15th (see #35), we are preparing e-mails for team leaders :)

**Comment by SadmanIslamAlif** [comment]
Thank you very much
